01-Authorization-RS256/server.js

```diff
@@ -27,7 +26,32 @@ const checkJwt = jwt({
   algorithms: ['RS256']
 });
 
-const checkScopes = jwtAuthz(['read:messages']);
+// const checkScopes = jwtAuthz(['read:messages']);
+const checkScopes = function (scopes) {
```

Comment:
@alexisluque why does the jwtAuthz middleware does not work?

Reply:
@aaguiarz the response code for the jwtAuthz package for the insufficient scope case is 401(unauthorized) but it should be 403

Reply:
I don't think is worth changing the Quickstart because of that. It makes it more complex. Can you submit a PR to https://github.com/auth0/express-jwt-authz to make it return 403?

Reply:
Done, you can find the PR [here](https://github.com/auth0/express-jwt-authz/pull/7)